integration/hive/src/main/java/org/apache/carbondata/hive/CarbonHiveRecordReader.java

```diff
@@ -106,6 +86,7 @@ private void initialize(InputSplit inputSplit, Configuration conf) throws IOExce
       int columnId = 0;
       for (int j = 0; j < arraySelectedColId.length; j++) {
         columnId = Integer.parseInt(arraySelectedColId[j]);
+
```

Comment:
revert this and remove local variable  `columnId`

Reply:
done